### PR TITLE
fix(oob): set connection alias when creating invitation

### PR DESCRIPTION
**Pull request by jakubkoci**
It was possible to set an alias when creating an invitation but it wasn't propagated to the connection created after receiving a connection request based on the invitation.

Signed-off-by: Jakub Koci <jakub.koci@gmail.com>

### CODE REVIEW

File: packages/core/src/storage/migration/__tests__/__snapshots__/0.1.test.ts.snap [hunk 1]
```diff
@@ -3132,443 +3132,3 @@ Object {
   },
 }
 `;
-
-exports[`UpdateAssistant | v0.1 - v0.2 should correctly update the role in the mediation record: allMediator 1`] = `
```
Comment (TimoGlastra) [inline packages/core/src/storage/migration/__tests__/__snapshots__/0.1.test.ts.snap:3136]: There's some snapshots being removed

**Review by TimoGlastra — COMMENTED** [review]
Nice! Isn't this a fix instead of a feature?

**Comment by jakubkoci** [comment]
> Nice! Isn't this a fix instead of a feature?

Yeah, maybe 🤷‍♂️ I'll change it.

**Review by TimoGlastra — APPROVED** [review]

**Comment by TimoGlastra** [comment]
Your last commit is not DCO signed off... :( Can you fix, then we can merge

**Pull request merged**